src/FlaUILibrary/flaui/module/element.py

```diff
@@ -87,13 +88,13 @@ def execute_action(self, action: Action, values: Container):
 
         switcher = {
             self.Action.FOCUS_ELEMENT:
-                lambda: self._get_element(values["xpath"]).Focus(),
+                lambda: self._focus_element(self._get_element(values["xpath"]), values["xpath"]),
             self.Action.GET_ELEMENT:
                 lambda: self._get_element(values["xpath"]),
             self.Action.GET_ELEMENT_BY_XPATH:
                 lambda: self._get_element_by_xpath(values["xpath"]),
             self.Action.FOCUS_ELEMENT_BY_XPATH:
-                lambda: self._get_element_by_xpath(values["xpath"]).Focus(),

```

Comment:
i will be deleting this function and action in my newer issue #163 because it will be obsolete 
So the only focus action will be available in line 90 as follow
self.Action.FOCUS_ELEMENT:
                lambda: self._focus_element(values["xpath"]),

Reply:
If you want to delete the function afterwards, you can do so ;) However, the PR is fine here.